src/vivarium_nih_us_cvd/data/loader.py

```diff
@@ -299,9 +309,9 @@ def get_entity(key: Union[str, EntityKey]):
 
 # Project-specific data functions
 def _load_and_sum_prevalence_from_sequelae(
-    key: str, map: Dict[str, List["Sequela"]], location: str
+    sequelae: List["Sequela"], location: str
```

Comment:
List[str]

Reply:
Eh, I'd just leave it - it happens a lot here. I must've copied/pasted old code so it might just be a way of more specifically specifying what type of string the list will consist of.